src/service/video-manager-impl.js

```diff
@@ -402,6 +405,10 @@ class VideoEntry {
       mutate,
     });
   }
+
+  handleReload() {
```

Comment:
Done. My original implementation was to avoid directly accessing a private method but they are in the same file. :man_shrugging: